### PR TITLE
This show how to fix the check

### PR DESCRIPTION
**Pull request by rocktavious**
DO NOT MERGE THIS IS A HOTFIX

## Issues

Because in 1.0 we didn't set the schema version to 1 if you go from 0.11.0 -> 1.2.x you are fine but if you created a new check predicate in 1.0 to < 1.2 you have a different schema for version 0 and this breaks when you jump up to 1.2.x 

This is a hot fix to 1.x line and we'll need to tell customers if they made checks in 1.0.x or 1.1.x they need to upgrade to 1.1.4 first before going to 1.2.x 😢 

TODO: tophatting and testing to ensure 1.0.x and 1.1.x can jump to 1.2 through this

### CODE REVIEW

File: opslevel/resource_opslevel_check_repository_grep.go [hunk 1]
```diff
@@ -113,6 +114,33 @@ func (r *CheckRepositoryGrepResource) Schema(ctx context.Context, req resource.S
...
+					"directory_search": schema.BoolAttribute{
+						Description: "Whether the check looks for the existence of a directory instead of a file.",
+						Required:    true,
+					},
+					"filepaths": schema.ListAttribute{
+						Description: "Restrict the search to certain file paths.",
+						Required:    true,
+						ElementType: types.StringType,
+					},
+					"file_contents_predicate": predicateSchema,
```
Comment (davidbloss) [inline opslevel/resource_opslevel_check_repository_grep.go:136]:
```suggestion
					"file_contents_predicate": PredicateSchema(),
```
I'm seeing `PredicateSchema()` in most places and assume this is what's needed?

Comment (rocktavious) [reply opslevel/resource_opslevel_check_repository_grep.go:136]: @davidbloss actually this one uses a local variable up on lines 118.  This was copied directly from 1.2

File: opslevel/resource_opslevel_check_repository_search.go [hunk 2]
```diff
@@ -118,6 +118,32 @@ func (r *CheckRepositorySearchResource) Schema(ctx context.Context, req resource
...
+				Attributes: CheckBaseAttributes(map[string]schema.Attribute{
+					"file_extensions": schema.SetAttribute{
+						Description: "Restrict the search to files of given extensions. Extensions should contain only letters and numbers. For example: [\"py\", \"rb\"].",
+						Optional:    true,
+						ElementType: types.StringType,
+						Validators: []validator.Set{
+							setvalidator.SizeAtLeast(1),
+						},
+					},
+					"file_contents_predicate": predicateSchema,
```
[the remark this answers is not in the document]
Comment (rocktavious) [reply opslevel/resource_opslevel_check_repository_search.go:139]: @davidbloss same as the other check

[the remark this answers is not in the document]
Comment (davidbloss) [reply opslevel/resource_opslevel_check_repository_search.go:139]: nvm, this was a special case

**Review by davidbloss — APPROVED** [review]
SHIP IT! 🚢 :shipit:

**Comment by rocktavious** [comment]
1.1.4 was released this code will never go into main as its only to fix 1.0.x and 1.1.x lines.

**Pull request closed without merging**